src/utils/components/AccessDenied/AccessDenied.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import restrictedSignImg from '@images/restricted-sign.svg';
+import { Alert } from '@patternfly/react-core';
+import { isEmpty } from '@utils/helpers';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+
+type AccessDeniedProps = {
+  message?: string;
+};
+
+const AccessDenied: React.FC<AccessDeniedProps> = ({ message }) => {
```

Comment:
Would you mind importing `FC` above like we do in the kubevirt-plugin repo? I think it's important to keep the same conventions in all repos we maintain.